packages/paypal-utils/README.md

```diff
@@ -0,0 +1,11 @@
+# paypal-utils
+
+This library was generated with [Nx](https://nx.dev).
```

Comment:
Can we please have a better description over here.

Reply:
@animesh1987 done